schema/url/visualizations.raml

```diff
@@ -28,20 +28,14 @@ types:
       value?: number
       units?: BinUnits
 
-  XAxisViewport:
+  NumericViewport:
     additionalProperties: false
     properties:
       xMin:
```

Comment:
If we want to keep NumericViewport pretty general, should probably have properties min and max instead of xMin and xMax? 

I recognize my suggestion may end up causing a lot of work changing xMin to min and so on, here and in plot.data. Since this is a b57 thing I'm also happy to make a new ticket for the name change and we can do it when time allows if you'd prefer?

Reply:
Yea it needs fixing in a number of repos and didn't seem urgent so I was planning to wait. New issue seems fine, but it probably is actually three tickets? Here, plot.data and web-eda I'd imagine at the least